lib/flv/index.js

```diff
@@ -1,4 +1,7 @@
+var transmuxer = require('./transmuxer');
+
 module.exports = {
   tag: require('./flv-tag'),
-  Transmuxer: require('./transmuxer')
+  Transmuxer: transmuxer.Transmuxer,
+  getFlvHeader: transmuxer.getFlvHeader
```

Comment:
Doing `transmuxer.Transmuxer` is confusing. I think it might be time to move `getFlvHeader` to it's own file. Then just import both of the files here.